### PR TITLE
Update theme to supply breakpoints to styled-system

### PR DESCRIPTION
**Pull request by ds300**
https://artsy.slack.com/archives/C2TQ4PT8R/p1534966253000100

Tested against reaction:

image

breakpoints

### CODE REVIEW

File: src/Theme.tsx [hunk 1]
```diff
@@ -31,6 +31,11 @@ const breakpoints = {
 export const themeProps = {
   borders: [0, "2px solid"],
 
+  /**
+   *  This allows styled-system to hook into our breakpoints
+   */
+  breakpoints: [breakpoints.sm, breakpoints.md, breakpoints.lg, breakpoints.xl],
```
Comment (damassi) [inline src/Theme.tsx]: 👍

**Comment by damassi** [comment]
LGTM 👍

**Comment by damassi** [comment]
@ds300 - Once the release goes out via Circle can you update Reaction with the latest version of palette?

**Comment by ds300** [comment]
Yep

File: src/Theme.tsx (hunk 1 above)
Comment (zephraph) [reply src/Theme.tsx]: This is missing xs... was that intentional?

Comment (zephraph) [reply src/Theme.tsx]: Neeeever mind. I see what it's doing. `xs` is just whatever is smaller than `sm`. 👍

**Pull request merged**